### DISCUSSION TITLE
Query model without loading permissions

### DISCUSSION THREAD

**Discussion by theisnygaard** [spatie/laravel-permission#2212] [Q&A]
Hello!

We have implemented the permission system in our Laravel which is working perfectly fine, but we're slowly seeing performance issues when handling large collections. 

We have a `User` model which uses the `HasRoles` and `HasPermissions` traits. In one of our views, we're showing a list of all users which has become quite slow. When inspecting the queries made to the database, the model seems to automatically load all permissions and roles related to that specific model (N+1), which is unnecessary at this point, since we're merely displaying meta data columns. 

Is it possible to query the model without loading the permissions? We've tried disabling the `roles` and `permissions` relationships via the `without` method, but with no luck.

Versions: Laravel `^8.0` and Spatie Laravel Permissions `^5.5`.

Thanks is advance.

Kind regards

**Comment by erikn69** [comment]
Hi, that is weird

Showing a list of all users do not bring any roles/permissions, it must be something on your implementation, try to replicate the problem on a fresh instalation and share it here 
https://spatie.be/docs/laravel-permission/v5/basic-usage/new-app#content-creating-a-demo-app
